platform/build-faiss/README.md

```diff
@@ -0,0 +1,57 @@
+# Set up an Ubuntu 22.04 machine to build FAISS
+
+## Setup for build in Ubuntu 22.04 with podman
+
+Add podman
+
+```sh
+sudo apt install podman -y
+```
+
+Run build in podman:
```

Comment:
I am familiar with `podman`. Dan Walsh is a great fella, although, with Red Hat's recent changes, it is not clear to me that we should accept a dependency on any single-vendor project, especially those sponsored directly by Red Hat.

Reply:
Good point. I personally would recommend [ociTools](https://ryantm.github.io/nixpkgs/builders/images/ocitools/) if we want to efficiently declare containers, which would let us rely on a much better-supported project with a more diverse user base. For now, I view Dockerfiles as a happy intermediate way to use either Docker or Podman, and we can always just remember to type "docker" on Docker-based machines.

Reply:
IMO, this is where its ↩ https://github.com/containerd/nerdctl

We could have continuous battles about what tools we use. These battle, ie. religious wars, are why (successful) tech firms build their own toolchain, top to bottom.

Are we, as a team of founding engineers, willing to adopt a pragmatic perspective?

Reply:
Sure. Pragmatically, containers run on k8s, Oracle, and Podman. That's it. Docker is a fading bad dream. For me, what runs the containers is uninteresting, thanks to OCI. And what builds the containers is also uninteresting, as long as it's reproducible; I can learn whatever builder you want to use.

Reply:
You state it well. We should, as a firm, attempt to eliminate ambiguity aversion. We do that by working on high-value areas. Packaging is not high value unless it directly relates to distribution.

Reply:
I think we understand our differing points of view. I also believe we all agree that `containerd` is the runtime we will use, if we need to run container workloads. All daemon (docker and a million clones including [cog](https://github.com/replicate/cog/blob/main/pkg/cli/run.go#L56). In addition there are rootless variants (podman, nerdctl, and many more).

`containerd` is one of many `OCI` runtimes. This runtime is widely used and has wide investment across the technology ecosystem. This component provides:

- Lifecycle Management (LCM):
  - start
  - stop
  - delete

- Runtime:
  - exec
  - logs

FWIIW,I agree, docker is and was a massive swing and a miss.